app/templates/buyers/brief_publish_confirmation.html

```diff
@@ -34,20 +34,39 @@
+
     {% if not published %}
-      <p class="padding-bottom-small">All requirements are published on the Digital Marketplace where anyone can see them.</p>
-      <p class="padding-bottom-small">All requirements are open for {{ dates.application_open_weeks }} weeks.</p>
-      <p class="padding-bottom-small">If you publish your requirements today ({{ dates.today | shortdateformat }}), suppliers will be able to apply until {{ dates.closing_time }}, {{ dates.closing_date | shortdateformat}}.</p>
+      <div class="dmspeak">
+        <p>All requirements are published on the Digital Marketplace where anyone can see them.</p>
+      </div>
+        {% if brief.lotSlug == 'digital-specialists' and not brief.requirementsLength %}
+          <p><a href="{{ url_for('.edit_brief_question', framework_slug=brief.frameworkSlug, lot_slug=brief.lotSlug, brief_id=brief.id, section_slug='set-how-long-your-requirements-will-be-live-for', question_id='requirementsLength') }}">Set how long your requirements will be open for</a></p>
+          <div class="dmspeak">
+            <p>This will show you what the supplier application deadline will be if you publish your requirements today.</p>
+          </div>
+        {% else %}
+          <p>Your requirements will be open for {{ dates.application_open_weeks }}.</p>
+          <div class="dmspeak">
+            <p>If you publish your requirements today ({{ dates.published_date | shortdateformat }}), suppliers will be able to apply until {{ dates.closing_time }}, {{ dates.closing_date | shortdateformat}}.</p>
+          </div>
+        {% endif %}
     {% endif %}
-    <p><strong>Supplier questions will be sent to:</strong></p>
-    <p class="padding-bottom-small">{{ email_address }}</p>
-    <p class="padding-bottom-small">Ensure this email address {% if not published %}will be{% else %}is{% endif %} monitored. If you’re away while suppliers can still ask questions, you should make sure your emails are forwarded to a colleague.</p>
+
+    <p>Supplier questions will be sent to:</p>
+    <div class="dmspeak">
+      <p>{{ email_address }}</p>
+    </div>
+    <div class="dmspeak">
+      <p>Make sure this email address {% if not published %}will be{% else %}is{% endif %} monitored. If you’re away while suppliers can still ask questions, you should make sure your emails are forwarded to a colleague.</p>
+    </div>
 
     {% if not published and unanswered_required > 0 %}
-      <p class="padding-bottom-small"><strong>You still need to complete the following questions before your requirements can be published:</strong></p>
+      <div class="dmspeak">
+        <p><strong>You still need to complete the following questions before your requirements can be published:</strong></p>
+      </div>
       {% for section in sections %}
         {% for question in section.questions %}
-          {% if question.answer_required %}
+          {% if question.answer_required and not question.id == 'requirementsLength' %}
```

Comment:
Yeah, that's what we're after. We decided repeating the message that they need to set the length looked odd. That's why in the view for this page there's a new bit of code about decreasing the unanswered questions count by one if requirementsLength isn't set.